index.html

```diff
@@ -1382,6 +1375,62 @@ <h2>
             </li>
           </ol>
         </section>
+        <section>
+          <h3>
+            Processing the manifest without a document
+          </h3>
+          <p>
+            The [=processing a manifest=] steps are invoked by [[HTML]]'s
+            processing steps for the [^link^] element, but MAY also be invoked
+            by the user agent to process a manifest without an associated
+            document.
+          </p>
+          <p>
+            In this case, the user agent SHOULD ensure that, at least at some
+            point in the past:
```

Comment:
```suggestion
            In this case, the user agent SHOULD ensure that at least at some
            point in the past:
```

Reply:
By the way, the "at least some point in the past" clause was to acknowledge that we don't need the user agent to verify this exactly at install time. It's sufficient to have verified this at the time you cached it (e.g. if you install on Machine A, you verify this, then sync the metadata to Machine B and install it, you don't need to verify it again on Machine B).

We thought about writing "at the time of fetching the metadata" but we wanted to also allow a situation where you verify it on Machine A, then you sync the manifest URL to Machine B, Machine B fetches the manifest but does not re-verify this condition. So we settled for the more permissive "at some point in the past".